decentraland/unity-explorer
# Qualify JS Code for unloading

## What does this PR change?

This change clears the scene JavaScript code cache when memory is low.

## How to test the changes?

1. Visit lots of scenes
2. Press the "low memory" button
3. See a handful of megabytes lower memory usage than without this change

The scenes visited must be the same for the comparison to be valid.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
